src/test/styles.test.js

```diff
@@ -191,4 +195,29 @@ describe('with styles', () => {
 
     Array.from(document.querySelectorAll('style')).forEach(el => expect(el.getAttribute('nonce')).toBe('foo'))
   })
+
+  it('should add a webpack nonce to the global style tags if one is available in the global scope', () => {
+    // eslint-disable-next-line no-underscore-dangle
+    window.__webpack_nonce__ = 'foo'
+
+    injectGlobal`
+        background: red;
+      `
```

Comment:
Can we make this a proper, valid example? E.g.

```JS
injectGlobal`
  html {
    background: red;
  }
`
```

Reply:
whoops, pushing the fix now